resources/views/admin.blade.php

```diff
@@ -70,5 +70,6 @@
     <script src="{{ asset('bower_components/review_travel/js_admin/demo/chart-pie-demo.js') }}"></script>
     <script src="{{ asset('bower_components/review_travel/vendor_adminvendor/jquery/jquery.min.js') }}"></script>
     <script src="{{ asset('bower_components/review_travel/vendor_adminvendor/bootstrap/js/bootstrap.bundle.min.js') }}"></script>
+    <script src="{{ asset('js/all.js') }}"></script>
```

Comment:
nếu đã mix rồi thì dùng như này nhé chị
```suggestion
    <script src="{{ mix('js/all.js') }}"></script>
```